internal/bminventory/inventory.go

```diff
@@ -2957,17 +2964,33 @@ func (b *bareMetalInventory) getHost(ctx context.Context, clusterId string, host
 	return &host, nil
 }
 
-func (b *bareMetalInventory) getCluster(ctx context.Context, clusterID string, returnHosts bool) (*common.Cluster, error) {
+func (b *bareMetalInventory) getCluster(
+	ctx context.Context,
+	clusterID string,
+	returnHosts bool,
```

Comment:
suggesting to create a type for this boolean, otherwise it's not clear in the usage 
something like `type withHosts bool`